core.R

```diff
@@ -10,7 +10,11 @@ prepare_dropdown_button <- function(refs_to_list = paste(
                                       "^latest-tag$",
                                       "^v([0-9]+\\.)?([0-9]+\\.)?([0-9]+)$",
                                       sep = "|"
-                                    )) {
+                                    ), versions_dropdownlist_configuration="") {
+  
+  if (!require("yaml")) install.packages("yaml")
+  config <- yaml::yaml.load(versions_dropdownlist_configuration)
```

Comment:
How much time does installing `yaml` take?

Reply:
I do not know. I am getting error like
```
Warning in install.packages("yaml") :
  'lib = "/usr/local/lib/R/site-library"' is not writable
  ```
Switching to native R-structure/code